dsn.go

```diff
@@ -169,6 +171,9 @@ func DSN(cfg *Config) (dsn string, err error) {
 	if cfg.ClientTimeout != defaultClientTimeout {
 		params.Add("clientTimeout", strconv.FormatInt(int64(cfg.ClientTimeout/time.Second), 10))
 	}
+	if cfg.JWTClientTimeout != defaultJWTClientTimeout {
+		params.Add("jwtClientTimeout", strconv.FormatInt(int64(cfg.JWTClientTimeout/time.Second), 10))
```

Comment:
should we validate negative timeout?

Reply:
I don't like adding in this PR. We have parameters cleanup in our roadmap for the upcoming future.